.agent/workflows/create_worktree.md

```
@@ -0,0 +1,36 @@
+---
+description: 新しい機能開発用のGit Worktreeを作成し、初期セットアップを行う手順
+---
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)